src/pages/staff/dashboard/VerificationPage.jsx

```
@@ -21,6 +24,178 @@ export default function VerificationPage() {
   const [error, setError] = useState('');
   const [metadata, setMetadata] = useState({});
   const [geolocationStatus, setGeolocationStatus] = useState('');
+  const [deviceLoc, setDeviceLoc] = useState(null);
+  const [storeInfo, setStoreInfo] = useState({ storeLoc: null, storeAddressText: '' });
```

ESLint

[error] 'storeInfo' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)